numpy/core/numeric.py

```diff
@@ -679,12 +679,12 @@ def correlate(a, v, mode='valid'):
     Cross-correlation of two 1-dimensional sequences.
 
     This function computes the correlation as generally defined in signal
-    processing texts::
+    processing texts:
 
-        c_{av}[k] = sum_n a[n+k] * conj(v[n])
+    .. math:: c_k = \sum_n a_{n+k} \cdot \overline{v_n}
```

Comment:
You need to change the `"""` to `r"""` a few lines above, to stop python consuming the \

Reply:
Ah, right of course! Thanks!